### PR TITLE
Save document title after editing it

### PR DESCRIPTION
**Pull request by elpoelma**
This ensures that the title of the document is saved just after having edited it. An additional save of the document is no longer required. Solves https://binnenland.atlassian.net/jira/software/c/projects/GN/boards/4?modal=detail&selectedIssue=GN-3734.

### CODE REVIEW

File: app/components/app-chrome.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ export default class AppChromeComponent extends Component {
   }
 
   @action
-  updateDocumentTitle(title) {
+  async updateDocumentTitle(title) {
```
Comment (abeforgit) [inline app/components/app-chrome.js:23]: why did this become async?

**Review by abeforgit — CHANGES_REQUESTED** [review]

**Comment by abeforgit** [comment]
Probably a good idea to make it reset on blur, since now if you edit the title but dont click the checkmark, it still looks like the title was changed both on the edit page and in the overview list, but if you refresh it reverts to the old title
ember-data has some kind of rollback method which will work in this case because the title is a simple property and not a relation

File: app/components/app-chrome.js (hunk 1 above)
Comment (elpoelma) [reply app/components/app-chrome.js]: I modified it to not be async anymore.

**Comment by elpoelma** [comment]
When we will refactor the AppChrome component, we should probably look into adding correct loading states/disabling buttons everywhere when saving/publishing a document or editing its title.

**Review by abeforgit — APPROVED** [review]

**Pull request merged**